src/plugins/condition/components/inspector/StyleEditor.vue

```diff
@@ -61,6 +61,13 @@
                                :options="isStyleInvisibleOption"
                                @change="updateStyleValue"
         />
+
+        <!-- Save Styles -->
+        <toolbar-button v-if="canSaveStyle()"
```

Comment:
lets make this a computed value, using methods, forces vue to call this function on every tick. But if its computed, it is optimized and only changes when the values referenced inside the computed function changes.

Reply:
good catch. done